ArduinoCode/Pedal_Potentiometers/Pedal_Potentiometers.ino

```diff
@@ -70,13 +70,24 @@ void readPotentiometers() {
   accelPin1Val = analogRead(accelPin1); 
   accelPin2Val = analogRead(accelPin2);
 
-  if (abs(accelPin1Val - accelPin2Val) <= (1024 * 0.1)) {
-    int averageReading = (accelPin1Val + accelPin2Val) / 2;
-    unsigned char byteReading = averageReading / 4;
-    unsigned char valueMessage[8] = {byteReading, 0, 0, 0, 0, 0, 0, 0};
-    CAN.sendMsgBuf(CAN_ACCEL, 0, 8, valueMessage); // send message of the average value of the two potentiometers
+  if (!invertorOn) {
+    return;
   } else {
```

Comment:
There's no need for the else as the return statement will exit the function and cause none of the code after it to run.